### ISSUE TITLE
Try to export image in PDF formatter with styling custom row column 

### ISSUE THREAD

**Issue by mosoksehbro** [dompdf/dompdf#3540]
Few issues with downloading, uploading media export with dompdf 

![image](https://github.com/user-attachments/assets/f000a766-a8ca-4cb0-b87b-6592d01772c1)

I use this version

![image](https://github.com/user-attachments/assets/3a8f9b38-2360-4f02-9b55-d61f4463f813)
Ini adalah aktual terjadi ketika mencoba untuk mendownload file archive, status pending sampai timeout responnya habis 

![image](https://github.com/user-attachments/assets/b0c4a56c-9992-4a7a-9d9d-e5add6c1bf00)
This is the config that I made for logic and use of dompdf

if you need more detail about this just tellme i will show what u need for more

**Comment by bsweeney** [comment]
Are you saying the browser just keep waiting and never displays the PDF?

**Comment by mosoksehbro** [comment]
> Are you saying the browser just keep waiting and never displays the PDF?

Yeah my browser just pending and never show display or action download if trying for download until respondtimeout

**Comment by bsweeney** [comment]
I don't have any insight into your framework (I don't use laravel) but you probably need to tell it you're done processing the request somehow. You might want to take your question to the [laravel project](https://github.com/barryvdh/laravel-dompdf)

**Comment by mosoksehbro** [comment]
> Saya tidak memiliki wawasan apa pun tentang kerangka kerja Anda (saya tidak menggunakan laravel), tetapi Anda mungkin perlu memberi tahu bahwa Anda telah selesai memproses permintaan tersebut. Anda mungkin ingin mengajukan pertanyaan Anda ke [proyek laravel](https://github.com/barryvdh/laravel-dompdf)

**Comment by mosoksehbro** [comment]
> Saya tidak memiliki wawasan apa pun tentang kerangka kerja Anda (saya tidak menggunakan laravel), tetapi Anda mungkin perlu memberi tahu bahwa Anda telah selesai memproses permintaan tersebut. Anda mungkin ingin mengajukan pertanyaan Anda ke [proyek laravel](https://github.com/barryvdh/laravel-dompdf)

I have done but still the same request still pending

**Comment by mosoksehbro** [comment]
// @extends('layouts.app')

@section('content')
    <div class="container">
        <h2>Archive</h2>
        @if (session('success'))
            <div class="alert alert-success">{{ session('success') }}</div>
        @endif

        
        <form method="POST" action="{{ route('archive.download') }}">
            @csrf
            <div class="row mb-3">
                <div class="col-md-4">
                    <label for="date_from">Date From</label>
                    <input type="date" name="date_from" id="date_from" class="form-control" required>
                </div>
                <div class="col-md-4">
                    <label for="date_to">Date To</label>
                    <input type="date" name="date_to" id="date_to" class="form-control" required>
                </div>
                <div class="col-md-4">
                    <label for="format">Format</label>
                    <select name="format" id="format" class="form-control" required>
                        <option value="">Select Format</option>
                        <option value="xlsx">XLSX</option>
                        <option value="pdf">PDF</option>
                    </select>
                </div>
            </div>
            <button type="submit" class="btn btn-primary">Download Archive</button>
        </form>

        <h3 class="mt-4">Media List</h3>
        <table class="table">
            <thead>
                <tr>
                    <th>Media</th>
                    <th>Created At</th>
                    <th>Caption</th>
                </tr>
            </thead>
            <tbody>
                @foreach ($mediaUploads as $media)
                    <tr>
                        <td>
                            @if (in_array($media->file_type, ['jpg', 'jpeg', 'png']))
                                <img src="{{ asset('storage/' . $media->file_path) }}" alt="media" width="100">
                            @elseif(in_array($media->file_type, ['mp4', 'mov']))
                                <video width="100" controls>
                                    <source src="{{ asset('storage/' . $media->file_path) }}"
                                        type="video/{{ $media->file_type }}">
                                    Your browser does not support the video tag.
                                </video>
                            @endif
                        </td>
                        <td>{{ $media->created_at->format('Y-m-d H:i:s') }}</td>
                        <td>{{ $media->caption }}</td>
                    </tr>
                @endforeach
            </tbody>
        </table>
    </div>
@endsection

**Comment by mosoksehbro** [comment]
```php


@extends('layouts.app')

@section('content')
    <div class="container">
        <h2>Archive</h2>
        @if (session('success'))
            <div class="alert alert-success">{{ session('success') }}</div>
        @endif

        <!-- Form Filter dan Download -->
        <form method="POST" action="{{ route('archive.download') }}">
            @csrf
            <div class="row mb-3">
                <div class="col-md-4">
                    <label for="date_from">Date From</label>
                    <input type="date" name="date_from" id="date_from" class="form-control" required>
                </div>
                <div class="col-md-4">
                    <label for="date_to">Date To</label>
                    <input type="date" name="date_to" id="date_to" class="form-control" required>
                </div>
                <div class="col-md-4">
                    <label for="format">Format</label>
                    <select name="format" id="format" class="form-control" required>
                        <option value="">Select Format</option>
                        <option value="xlsx">XLSX</option>
                        <option value="pdf">PDF</option>
                    </select>
                </div>
            </div>
            <button type="submit" class="btn btn-primary">Download Archive</button>
        </form>

        <h3 class="mt-4">Media List</h3>
        <table class="table">
            <thead>
                <tr>
                    <th>Media</th>
                    <th>Created At</th>
                    <th>Caption</th>
                </tr>
            </thead>
            <tbody>
                @foreach ($mediaUploads as $media)
                    <tr>
                        <td>
                            @if (in_array($media->file_type, ['jpg', 'jpeg', 'png']))
                                <img src="{{ asset('storage/' . $media->file_path) }}" alt="media" width="100">
                            @elseif(in_array($media->file_type, ['mp4', 'mov']))
                                <video width="100" controls>
                                    <source src="{{ asset('storage/' . $media->file_path) }}"
                                        type="video/{{ $media->file_type }}">
                                    Your browser does not support the video tag.
                                </video>
                            @endif
                        </td>
                        <td>{{ $media->created_at->format('Y-m-d H:i:s') }}</td>
                        <td>{{ $media->caption }}</td>
                    </tr>
                @endforeach
            </tbody>
        </table>
    </div>
@endsection
```

**Comment by bsweeney** [comment]
So, this appears to be an issue where the document is never fully streamed to the end user. Typically this happens when an intermediate system is expecting more data that never comes. You might try retrieving the PDF content directly instead of passing back the response from laravel-dompdf. Or saving the file and redirecting the user to the saved file.

Another potential issue, since you're using tables, is that the render never actually completes. A table cell that is larger than a single page can cause this. Your sample templates doesn't appear to indicate this particular issue, but it's worth checking in case one of your images is much taller than wider.

**Comment by mosoksehbro** [comment]
I have tried and almost solved this problem by using embed64 but a new problem appears not all resolutions support using this embed 64 method

**Comment by bsweeney** [comment]
Base64 images could be running into a parsing issue if the size is large. Next release should address that scenario.
